iothub/service/tests/DigitalTwin/UpdateOperationsUtilityTests.cs

```diff
@@ -0,0 +1,158 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using FluentAssertions;
+using Microsoft.Azure.Devices.Serialization;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.Devices.Tests.DigitalTwin
+{
+    [TestClass]
+    [TestCategory("Unit")]
+    public class UpdateOperationsUtilityTests
+    {
+        private const string Op = "op";
+        private const string Add = "add";
+        private const string Replace = "replace";
+        private const string Remove = "remove";
+        private const string Value = "value";
+        private const string Metadata = "$metadata";
+
+        [TestMethod]
+        public void UpdateUtilityAppendsAddPropertyOp()
+        {
+            UpdateOperationsUtility op = new UpdateOperationsUtility();
```

Comment:
Use var here as per coding guidelines as the right hand side has the type.

Reply:
good catch, will update.